api/src/auth/drivers/openid.ts

```diff
@@ -178,6 +197,31 @@ export class OpenIDAuthDriver extends LocalAuthDriver {
 			throw handleError(e);
 		}
 
+		let role = this.config['defaultRoleId'];
+		const groupClaimName: string = this.config['groupClaimName'] ?? 'groups';
+
+		if (userInfo[groupClaimName] === undefined) {
```

Comment:
This seems aggressively non-optional. I would say "groups" should be an opt-in feature, since the majority of OIDC providers provide no such data.

Reply:
Oh right, I didn't even think about this :facepalm:. Thanks for pointing that out!